frontend/talentsearch/src/js/components/workLocationPreferenceForm/WorkLocationPreferenceForm.test.tsx

```diff
@@ -0,0 +1,211 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from "react";
+import "@testing-library/jest-dom";
+import { IntlProvider, MessageFormatElement } from "react-intl";
+import { User } from "@common/api/generated";
+import { render, screen, fireEvent, act } from "../../tests/testUtils";
+import { WorkLocationPreferenceForm } from "./WorkLocationPreferenceForm";
+
+const renderWithReactIntl = (
+  component: React.ReactNode,
+  locale?: "en" | "fr",
+  messages?: Record<string, string> | Record<string, MessageFormatElement[]>,
+) => {
+  return act(() => {
+    render(
+      <IntlProvider locale={locale || "en"} messages={messages}>
+        {component}
+      </IntlProvider>,
+    );
+  });
+};
+
+const user: User = {
+  id: "1",
+  firstName: "Maura",
+  lastName: "Attow",
+  email: "mattow0@ning.com",
+  telephone: "+867365373244",
+};
+
+const onClick = jest.fn();
+
+const renderWorkLocationPreference = () => {
+  return renderWithReactIntl(
+    <WorkLocationPreferenceForm
+      initialData={user}
+      handleWorkLocationPreference={onClick}
+    />,
+  );
+};
+
+describe("Work Location Preference Form tests", () => {
```

Comment:
Don't forget to finish the commented out tests. :-)

Reply:
Done